OpenBCI Software/01-OpenBCI_GUI.md

```diff
@@ -117,36 +134,20 @@ Then double click the executable to run it for the first time. You must check bo
 
 ![accept networking](../assets/images/gui_windows_accept_networking.PNG)
 
-We recommend you leave the Hub running, even when not using the OpenBCI_GUI, the hub's impact on performance when not in use is minimal. As of GUI v4, the GUI will auto-launch the Hub on Windows if both apps are set to 'Run this program as an administrator'.
+We recommend you leave the Hub running, even when not using the OpenBCI_GUI, the hub's impact on performance when not in use is minimal.
```

Comment:
Since the HUB runs automatically from inside the GUI now, there's no need to leave the hub running